docs/installation/deployment/kubernetes.rst

```diff
@@ -1,5 +1,30 @@
 .. _deployment_kubernetes:
 
-==============================
-Deploying on Kubernetes (TODO)
-==============================
+==========
+Kubernetes
+==========
+
+Here you can find a reference implementation of a Open Notificaties deployment for
```

Comment:
Let's also refer to https://github.com/maykinmedia/charts/tree/main/charts/opennotificaties in this file